pages/eventscouting.tsx

```diff
@@ -36,9 +36,18 @@ const EventScouting: NextPage = () => {
 
             {
               scoutingConfig[index].inputs.map((element, index) => {
-                  return (
-                    <input type={element.type} placeholder={element.name} key={index}/>
-                  );
+                  if (element.type === "number") {
+                      return (
+                        <input type={element.type} placeholder={element.name} key={index}/>
+                      );
+                  } else {
+                      return (
+                        <div>
```

Comment:
using divs and span elements isn't usually best practice because it doesn't give screen readers/browsers information on what is in the elements